src/processes/domain-process.js

```diff
@@ -37,7 +37,7 @@ const postToGetstream = async (activity) => {
       returnActivityId: activityId
     };
   } catch (error) {
-    console.info(error);
+    console.info('error', error);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logging statement has been updated to provide more context about the error. This is a good practice as it helps with debugging and understanding the nature of the error.
```diff
- console.info(error);
+ console.info('error', error);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->